crew

```diff
@@ -69,6 +69,49 @@ def search (pkgName, silent = false)
   abort "package #{pkgName} not found :("
 end
 
+def help (pkgName)
+  case @pkgName
```

Comment:
you can write `case pkgName` here.  @pkgName is an instance variable.  And pkgName is a local variable.